README.rst

```diff
@@ -50,6 +50,58 @@ can optionally be installed together and used as ``torchpme.metatensor`` via
 
     pip install torch-pme[metatensor]
 
+.. marker-quickstart
+
+Quickstart
+----------
+
+Here is a simple example get you started with *torch-pme*:
+
+.. code-block:: python
+
+   import torch
+   import torchpme
+
+   # Single charge in a cubic box
+   positions = torch.zeros((1, 3), requires_grad=True)
+   cell = 8 * torch.eye(3)
+   charges = torch.tensor([[1.0]])
+
+   # No neighbors for a single atom; use `vesin` for neighbors if needed
```

Comment:
If it would be better to say how to use vesin here?

Reply:
Good point and I thought about it as well. In the end I decided against to keep the quickstart short and only about torch-pme; details will be shown in the basic usage tutorial.

Reply:
This is a good point though. I added some comments in `basic-usage.py` to explain why we use the neighbor list from `vesin` instead of some other package.